nodemon.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ignore": ["db/*"]
```

Comment:
Can we pass this info as arguments to the npm script instead of a separate file.

In general, using files for configs is a good practice. However, in this case we're only talking about development environment configs, which do not necessarily impact app behaviour

Reply:
I agree. The alternative is to change the "start" script to ``` "start": "nodemon src/main.js" ```.